scripts.js

```diff
@@ -52,3 +52,47 @@ container.addEventListener('click', (e) => {
     book.remove(index);
   }
 });
+
+const listLink = document.querySelector('#list-link');
+const addBookLink = document.querySelector('#addbook-link');
+const contactLink = document.querySelector('#contact-link');
+
+const bookListSection = document.querySelector('#booklist-section');
+const addbookInputs = document.querySelector('#addbook-inputs');
+const contactSection = document.querySelector('#contact-section');
+
+listLink.addEventListener('click', () => {
+  bookListSection.classList.add('visible');
+  addbookInputs.classList.add('hidden');
+  contactSection.classList.add('hidden');
+  addbookInputs.classList.remove('visible');
+  contactSection.classList.remove('visible');
+});
+
+addBookLink.addEventListener('click', () => {
+  bookListSection.classList.remove('visible');
+  contactSection.classList.remove('visible');
+  bookListSection.classList.add('hidden');
+  contactSection.classList.add('hidden');
+  addbookInputs.classList.add('visible');
+});
+contactLink.addEventListener('click', () => {
+  addbookInputs.classList.add('hidden');
+  bookListSection.classList.add('hidden');
+  contactSection.classList.add('visible');
+  addbookInputs.classList.remove('visible');
+  bookListSection.classList.remove('visible');
+});
+
+const dateContainer = document.querySelector('.top-info');
+const date = new Date();
+dateContainer.innerHTML = `<p><span class="date">${date.toLocaleString('en-US', { month: 'long' })} ${date.getDay()} ${date.getFullYear()}</span>, <span class="time">${date.toLocaleTimeString()}</span></p>`;
+
+/* eslint-disable */
```

Comment:
- [ ] It will be great to remove the `/* eslint-disable */` from your code base and instead fix the error that it presents. Eslint is great to catching error and enables one to write clean code that has no errors